### ISSUE TITLE
Names of main tables are not announced to VoiceOver

### ISSUE THREAD

**Issue by PeterSaathoff-Harshfield** [brentsimmons/NetNewsWire#934]
When VoiceOver's focus lands on the sidebar and articles table, the names of these tables are not read by VoiceOver.

Reproduction steps:
1. Enable VoiceOver (cmd F5)
2. Linear navigate to the sidebar (control + option + right/left arrows)
3. Observe that the sidebar is reported as something like, "table Today row 2 selected"
4. Linear navigate to the articles table
5. Observe that the articles table is reported as simply, "table" unless an article is selected, in which case the selected article's title follows

Note: In Mail.app, the word "mailboxes" precedes the word table for the sidebar, and the word "messages" precedes the word table for the messages list.

Expected: The sidebar is labeled "feeds" and the articles list is labeled "articles."

**Comment by vincode-io** [comment]
The Feeds, Timeline, and Article panes have had accessibility labels added so that they are announced as such.  This will be in the next 5.0.1 release.
